### PR TITLE
[release-4.14] OCPBUGS-20472: hosted cluster upgrade failure from 4.13 stable to 4.14 

### PR DESCRIPTION
**Pull request by flavio-fernandes**
In Hypershift deployments, pods in management cluster should not attempt to create a
directory for separate logs. That would happen in single-zone and zingle-zone-tmp phases,
since connectivity to ovn db is still in use from master pods.

To keep it consistent, this change is also being applied to the self-hosted configuration.

### CODE REVIEW

**Comment by flavio-fernandes** [comment]
```
rehearse-44190-periodic-ci-openshift-openshift-tests-private-release-4.14-amd64-nightly-4.14-upgrade-from-stable-4.13-aws-ipi-ovn-hypershift-replace-f7

periodic-ci-openshift-openshift-tests-private-release-4.14-amd64-nightly-4.14-upgrade-from-stable-4.13-aws-ipi-ovn-hypershift-replace-f7
```

File: bindata/network/ovn-kubernetes/self-hosted/single-zone-interconnect/004-config.yaml [hunk 1]
```diff
@@ -0,0 +1,87 @@
+---
```
Comment (flavio-fernandes) [inline bindata/network/ovn-kubernetes/self-hosted/single-zone-interconnect/004-config.yaml:1]: **Note**: the only difference between this file and the one in multi-zone in self-hosted is that it lacks the configuration for libovsdb logs.

File: bindata/network/ovn-kubernetes/managed/single-zone-interconnect/004-config.yaml [hunk 2]
```diff
@@ -0,0 +1,162 @@
+---
```
Comment (flavio-fernandes) [inline bindata/network/ovn-kubernetes/managed/single-zone-interconnect/004-config.yaml:1]: **Note**: the only difference between this file and the one in multi-zone in managed is that it lacks the configuration for libovsdb logs.

**Comment by flavio-fernandes** [comment]
/hold

Need to test job, which is being made avaialbe to CNO by @jluhrsen via PR https://github.com/openshift/release/pull/44320

**Comment by npinaeva** [comment]
makes sense! let's see what the test says

**Comment by flavio-fernandes** [comment]
Closing this. We are going to use https://github.com/openshift/cluster-network-operator/pull/2063 to address this issue.

**Pull request closed without merging**